### PR TITLE
Update task-configuration-file-full.md

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Added new configurations for TiDB Lightning:  import-mode,  on-dupicate

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/8224
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by glorv — APPROVED** [review]

**Review by sunzhaoyang — APPROVED** [review]
LGTM

File: dm/task-configuration-file-full.md [hunk 1]
```diff
@@ -111,6 +111,16 @@ loaders:
...
 
+# The import mode during the full import phase. You can set it to the following:
+# - "sql" (default). Use [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md) TiDB-backend mode to import data.
+# - "loader". Use Loader mode to import data. This mode is only for compatibility with features that TiDB Lightning does not support yet. It will be deprecated in the future.
+import-mode: "sql"
+#  Methods to resolve conflicts during the full import phase. You can set it to the following:
+# - "replace" (default). Only supports the import mode "sql". In this method, it uses the new data to replace the existing data.
+# - "ignore". Only supports the import mode "sql". It keeps the existing data, and ignores the new data.
+# - "error". Only supports the import mode "loader". It reports errors when inserting duplicated data, and then stops the replication task.
+on-dupicate: "replace"
```
Comment (ran-huang) [inline dm/task-configuration-file-full.md:122]: Please indent as the Chinese version does.

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

File: dm/task-configuration-file-full.md (hunk 1 above)
Comment (hfxsd) [reply dm/task-configuration-file-full.md:122]: Done

File: dm/task-configuration-file-full.md [hunk 2]
```diff
@@ -109,10 +109,18 @@ mydumpers:
 loaders:
   global:                            # The configuration name of the processing unit.
     pool-size: 16                    # The number of threads that concurrently execute dumped SQL files in the load processing unit (16 by default). When multiple instances are migrating data to TiDB at the same time, slightly reduce the value according to the load.
-
     # The directory that stores full data exported from the upstream ("./dumped_data" by default).
     # Supoprts a local filesystem path or an Amazon S3 path. For example, "s3://dm_bucket/dumped_data?region=us-west-2&endpoint=s3-website.us-east-2.amazonaws.com&access_key=s3accesskey&secret_access_key=s3secretkey&force_path_style=true"
     dir: "./dumped_data"
+    # The import mode during the full import phase. You can set it to the following:
```
Comment (sunzhaoyang) [inline dm/task-configuration-file-full.md:115]:
```suggestion
    # The import mode during the full import phase. In most cases you don't need to care about this configuration. 
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**